src/emulator/functionsEmulator.ts

```diff
@@ -500,6 +547,116 @@ You can probably fix this by running "npm install ${
   getName(): Emulators {
     return Emulators.FUNCTIONS;
   }
+
+  getTriggers(): EmulatedTriggerDefinition[] {
+    return this.triggers;
+  }
+
+  /**
+   * Returns the path to a "node" executable to use.
+   */
+  async askInstallNodeVersion(cwd: string): Promise<string> {
```

Comment:
This method was not changed, it was just moved to be inside the `FunctionsEmulator` class rather than floating.